app/helpers/utils.py

```diff
@@ -36,3 +45,58 @@ def get_logging_cfg():
 def init_logging():
     config = get_logging_cfg()
     logging.config.dictConfig(config)
+
+
+def prevent_erroneous_kml(kml_string):
+    # remove all attributes with on prefix and all script elements
+    kml_string = re.sub(r'on\w*=(".+?"|\'.+?\')', '', kml_string, flags=re.IGNORECASE)
+    kml_string = re.sub(
+        r'(<|&lt;)\s*\bscript\b.*?(>|&gt;).*?(<|&lt;)/\s*\bscript\b\s*(>|&gt;)',
+        ' ',
+        kml_string,
+        flags=re.IGNORECASE | re.DOTALL
+    )
+    return kml_string
+
+
+def bytes_to_mb(bytes, to, b_size=1024):
+    a = {'k': 1, 'm': 2, 'g': 3, 't': 4, 'p': 5, 'e': 6}
+    r = float(bytes)
+    return bytes / (b_size**a[to])
```

Comment:
Nice methods however I would change it as follow:

```python
def bytes_to(bytes, to, b_size=1024):
    a = {'kb': 1, 'mb': 2, 'gb': 3, 'tb': 4}
    r = float(bytes)
    return bytes / (b_size**a[to.lower()])
```
This made the usage clearer ;-) also peta und exa bytes are not needed.